packages/schema/breadboard.schema.json

```diff
@@ -6,7 +6,7 @@
   "$defs": {
     "identifier": {
       "type": "string",
-      "pattern": "^[a-zA-Z_][a-zA-Z0-9_-]*$"
+      "pattern": "^[\\$a-zA-Z_][a-zA-Z0-9_-]*$"
```

Comment:
```suggestion
      "pattern": "^(\\$error)|[a-zA-Z_][a-zA-Z0-9_-]*$"
```

Reply:
I know the parentheses at the start are superfluous, it just makes it a bit more clear